kubernetes/resource_kubernetes_runtime_class_v1.go

```diff
@@ -0,0 +1,177 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package kubernetes
+
+import (
+	"context"
+	"log"
+	"regexp"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	nodev1 "k8s.io/api/node/v1"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	pkgApi "k8s.io/apimachinery/pkg/types"
+)
+
+func resourceKubernetesRuntimeClassV1() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceKubernetesRuntimeClassV1Create,
+		ReadContext:   resourceKubernetesRuntimeClassV1Read,
+		UpdateContext: resourceKubernetesRuntimeClassV1Update,
+		DeleteContext: resourceKubernetesRuntimeClassV1Delete,
+
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"metadata": metadataSchema("runtimeclass", true),
+
```

Comment:
What about the `overhead` and `scheduling` fields? See API ref: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.25/#runtimeclass-v1-node-k8s-io

Reply:
Sacha had recommended that i focus on the handler at first and get it working perfectly, he also mentioned that since the handler is will be the most commonly used we should merge this first and then add `overhead` and `scheduling` afterwards